wiki/Help_Centre/Account_Restrictions/id.md

```diff
@@ -18,9 +15,9 @@ Fitur berikut dinonaktifkan untuk akun yang dibatasi:
 
 ## Akun saya telah dibatasi! Apa yang harus saya lakukan?
 
-Jika akun Anda telah dibatasi, jangan panik. Silakan ikuti langkah-langkah di bawah ini:
```

Comment:
not sure why this was removed. seems to match english (needs a newline after though)